src/interaction/InteractionManager.js

```diff
@@ -414,12 +436,45 @@ InteractionManager.prototype.processMouseDown = function ( displayObject, hit )
 
     if(hit)
     {
-        displayObject[ isRightButton ? '_isRightDown' : '_isLeftDown' ] = true;
-        this.dispatchEvent( displayObject, isRightButton ? 'rightdown' : 'mousedown', this.eventData );
+        // only when the double click callback is defined we should perform
+        // the logic for checking double clicks
+        if(displayObject.dblclick)
+        {
+            this._clickCount++;
+
+            if(this._clickCount === 1)
+            {
+                this._dblClickTimer = setTimeout(function() {
+                    displayObject[ isRightButton ? '_isRightDown' : '_isLeftDown' ] = true;
+                    this.dispatchEvent( displayObject, isRightButton ? 'rightdown' : 'mousedown', this.eventData );
+                    this._clickCount = 0;
+                }.bind(this), displayObject.dblClickDelay);
```

Comment:
creating a bound function every mouse event will be very expensive. Use a scope variable for context instead.